swagger.yaml

```diff
@@ -3876,6 +3881,11 @@ definitions:
         description: List of OLM operators to be installed.
         items:
           $ref: '#/definitions/operator-create-params'
+      hypethreading_config:
+        type: string
+        description: indicate whether hyperthreading needs to be activate on masters, workers, both.
+        enum: ['Masters', 'Workers', 'All']
+        x-nullable: true
```

Comment:
Copy from above, but here we also need none, right?

Reply:
fixed